project1/css/index.css

```diff
@@ -62,3 +62,102 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+
+.logo{
+	position:absolute;
+	top:0px;
+	right:0px;
+	float:right;
+	padding:25px 15% 22px 30px;
+}
+
+nav{
+	padding:35px;
+	text-align: justify;
+	width:90%;
+}
+
+nav li{
+	display:inline;
+	font-size:1.05em;
+	padding-right: 40px;
+	padding-left:20px;
+}
+
+nav a{
+	color:gray;
+	text-decoration: none;
+}
+.cta button{
+	position:relative;
+	top:-30px;
+	left:10px;
+	background-color:white;
+	color:black;
+	border:solid 1px;
+	font-size:22%;
+	text-align: center;
+	padding: 10px 35px 10px 35px;
+}
+.cta{
+	width:900px;
+	padding:10px 20px 30px 30px;
+}
+.cta-text{
+	position: relative;
+	top:-40px;
+	right:-100px;
+	display:inline-block;
+	text-align: center;
+	justify-content: center;
+	font-size:480%;
+	padding:0px;
+}
+#splashimg{
+	display:inline-block;
+	padding:0px;
+	position: relative;
+	right:-20%;
+}
+.main-content{
+	align-content: center;
+	width:900px;
+	padding:0px 0px 30px 30px;
+	display:inline-block;
+}
+.main-content p{
+	
+	font-size:18px;
+}
+.main-content h4{
+	font-size:20px;
+}
+.top-content .text-content{
+	display:inline-block;
+	width:432px;
+	padding:20px;
+}
+.middle-img{
+	padding:10px;
+}
+.bottom-content{
+	padding:0px 0px 5px 20px;
+}
+.bottom-content .text-content{
+	display:inline-block;
+	width:33%;
+	padding:10px;
+}
+.contact{
+	padding:0px 0px 30px 30px;
+}
+.contact h4{
+	padding:5px;
+}
+.contact p{
+	padding:10px;
+}
+footer{
+	width:100%;
+	text-align: center;
+}
```

Comment:
You can center everything by creating a selector for your container class and adding a large fixed width -- setting it, for example, to 1000 pixels -- and then setting the margin to 0.